libs/designsystem/src/lib/components/modal/services/modal.interfaces.ts

```diff
@@ -12,6 +12,7 @@ export interface Overlay {
 
 export abstract class Modal {
   didPresent: Promise<void>;
+  willDismiss: Promise<void>;
```

Comment:
I know this is the Ionic jargon, but maybe align with existing Kirby wording?
```suggestion
  willClose: Promise<void>;
```

Reply:
done

Reply:
done